File: henripoitierfe/src/app/books/book.ts
export class Book {



  private _isbn: string;
  private _title: string;
  private _price: string;
  private _cover: string;
  private _synopsis: string[];
  private _qte:number;

  get qte(): number {
    return this._qte;
  }

  set qte(value: number) {
    this._qte = value;
  }

  set isbn(value: string) {
    this._isbn = value;
  }

  get title(): string {
    return this._title;
  }

  set title(value: string) {
    this._title = value;
  }

  get price(): string {
    return this._price;
  }

  set price(value: string) {
    this._price = value;
  }

  get cover(): string {
    return this._cover;
  }

  set cover(value: string) {
    this._cover = value;
  }

  get synopsis(): string[] {
    return this._synopsis;
  }

  set synopsis(value: string[]) {
    this._synopsis = value;
  }
}

File: henripoitierbe/src/main/java/com/beezen/henripoitierbe/entity/Book.java
package com.beezen.henripoitierbe.entity;

import javax.persistence.*;
import java.io.Serializable;

@Entity
@Table(name = "books")
public class Book implements Serializable {

    @Id
    private String isbn;
    private String title;
    private Double price;
    private String cover;
    @Column(length=10000)
    private String[] synopsis;


    public String getIsbn() {
        return isbn;
    }

    public void setIsbn(String isbn) {
        this.isbn = isbn;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public Double getPrice() {
        return price;
    }

    public void setPrice(Double price) {
        this.price = price;
    }

    public String getCover() {
        return cover;
    }

    public void setCover(String cover) {
        this.cover = cover;
    }

    public String[] getSynopsis() {
        return synopsis;
    }

    public void setSynopsis(String[] synopsis) {
        this.synopsis = synopsis;
    }
}
